lib/puppet/face/facts.rb

```diff
@@ -2,19 +2,19 @@
 require 'puppet/node/facts'
 require 'puppet/util/fact_dif'
 
-EXCLUDE_LIST = %w[ facterversion
-  load_averages\..*
-  processors\.speed
-  swapfree swapfree_mb
-  memoryfree memoryfree_mb
-  memory\.swap\.available_bytes memory\.swap\.used_bytes
-  memory\.swap\.available memory\.swap\.capacity memory\.swap\.used
-  memory\.system\.available_bytes memory\.system\.used_bytes
-  memory\.system\.available memory\.system\.capacity memory\.system\.used
-  mountpoints\..*\.available.* mountpoints\..*\.capacity mountpoints\..*\.used.*
-  sp_uptime system_profiler\.uptime
-  uptime uptime_days uptime_hours uptime_seconds
-  system_uptime\.uptime system_uptime\.days system_uptime\.hours system_uptime\.seconds
+EXCLUDE_LIST = %w[ ^facterversion$
```

Comment:
This is changed to avoid breaking regex given by users via `--exclude` CLI option.